### PR TITLE
Ats/landingpage

### PR DESCRIPTION
**Pull request by arshtsingh**

### CODE REVIEW

File: frontend/src/components/home/SignUp.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          className="shadow appearance-none border w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+        />
+      </label>
+      <label>
+        <input
+          type="email"
+          placeholder={"your best email"}
+          onChange={(e) => setEmail(e.target.value)}
+          value={email}
+          className="shadow appearance-none border  w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
```
Comment (enricozb) [inline frontend/src/components/home/SignUp.tsx]: double space in here

File: frontend/src/components/home/SignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  const [email, setEmail] = useState("")
+
+  const submit = async (event: React.SyntheticEvent) => {
+    event.preventDefault()
+    try {
+      const emailEntry = { email, name }
+      await axios.post("/api/user/waitlist", emailEntry)
+      setShowForm(false)
+      toast("Successfully added to the waitlist!")
+    } catch (e) {
```
Comment (enricozb) [inline frontend/src/components/home/SignUp.tsx:18]: can you log the error as well pls

**Review by enricozb — APPROVED** [review]

File: frontend/src/components/home/MarketingContent.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          style: {
+            background: "#FFFFFF",
+          },
+        }}
+      />
+      <img
+        className="object-scale-down h-96 "
+        src="https://nickpassey.com/wp-content/uploads/2016/10/Featured.jpg"
+        alt=""
+      ></img>
```
Comment (enricozb) [inline frontend/src/components/home/MarketingContent.tsx]: Please remove this

File: frontend/src/components/home/SignUp.tsx (hunk 1 above)
Comment (arshtsingh) [reply frontend/src/components/home/SignUp.tsx]: Fixed in latest commit

File: frontend/src/components/home/SignUp.tsx (hunk 2 above)
Comment (arshtsingh) [reply frontend/src/components/home/SignUp.tsx:18]: Added logging the error

File: frontend/src/components/home/MarketingContent.tsx (hunk 3 above)
Comment (arshtsingh) [reply frontend/src/components/home/MarketingContent.tsx]: placeholder for the time being

Comment (arshtsingh) [reply frontend/src/components/home/MarketingContent.tsx]: removed

**Pull request merged**